test/models/admin_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class AdminTest < ActiveSupport::TestCase
+  test 'create' do
+    admin = create(:admin)
+    assert admin.persisted?
+  end
+end
```

Comment:
You can remove all the model tests, we are not using them. The code will be tested via endpoints - controllers.